src/utils/ledgerHash.ts

```diff
@@ -45,8 +45,9 @@ function computeTransactionHash(
       )
     } catch (e) {
       if (
+        e instanceof Error &&
         e.toString() ===
-        'SyntaxError: Unexpected' + ' token u in JSON at position 0'
+          'SyntaxError: Unexpected' + ' token u in JSON at position 0'
```

Comment:
While you're here, can you make this a little cleaner?
```suggestion
          'SyntaxError: Unexpected token u in JSON at position 0'
```